app/models/physical_test.rb

```diff
@@ -0,0 +1,12 @@
+class PhysicalTest < ApplicationRecord
+  belongs_to :user
+  has_many :tests
+
+  enum suggested_program: {
+    UBP: 1,
```

Comment:
Should we avoid using the acronyms? I think it can lead to misunderstandings, confusion and unreadability.